sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -771,6 +771,31 @@ class QueryExecutionErrorsSuite
     assert(e.getErrorClass === "STREAM_FAILED")
     assert(e.getCause.isInstanceOf[NullPointerException])
   }
+
+  test("UNSUPPORTED_EXPR_FOR_WINDOW: to_date is not supported with WINDOW") {
```

Comment:
Does it also need JIRA number here?

Reply:
It is optional. Usually, we add an JIRA number to refer a bug report that the test addresses. JIRA numbers don't bring additional info for regular tests like this.